src/cryptopp.mk

```diff
@@ -0,0 +1,38 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := cryptopp
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 5.6.3
+$(PKG)_CHECKSUM := 9390670a14170dd0f48a6b6b06f74269ef4b056d4718a1a329f6f6069dc957c9
+$(PKG)_SUBDIR   := 
+$(PKG)_VERSIONF := $(shell echo $($(PKG)_VERSION) | tr -d .)
+$(PKG)_FILE     := $(PKG)$($(PKG)_VERSIONF).zip
+$(PKG)_URL      := http://prdownloads.sourceforge.net/cryptopp/$($(PKG)_FILE)
+$(PKG)_URL_2    := https://www.cryptopp.com/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'https://www.cryptopp.com/' | \
+    $(SED) -n 's,<TITLE>Crypto++ Library \([0-9]\.[0-9]\.[0-9]\).*,\1,p'
+endef
+
+define $(PKG)_BUILD
+    cp $(1)config.recommend $(1)config.h
+    $(MAKE) -C '$(1)' -j '$(JOBS)' -f GNUmakefile \
+	    CC=$(TARGET)-gcc \
+	    CXX=$(TARGET)-g++ \
+	    RANLIB=$(TARGET)-ranlib \
+	    AR=$(TARGET)-ar \
+	    LD=$(TARGET)-ld \
+	    CXXFLAGS='-DNDEBUG -O3 -mtune=native -pipe' \
+	    $(if $(BUILD_STATIC),libcryptopp.a,cryptopp.dll)
+
+    mkdir -p $(PREFIX)/$(TARGET)/include/cryptopp
+    chmod 755 $(PREFIX)/$(TARGET)/include/cryptopp
+    cp $(1)*.h $(PREFIX)/$(TARGET)/include/cryptopp
+    chmod 644 $(PREFIX)/$(TARGET)/include/cryptopp/*
+    cp $(1)$(if $(BUILD_STATIC),libcryptopp.a,cryptopp.dll) $(PREFIX)/$(TARGET)/lib
```

Comment:
- `dll` goes to `bin/`
- `.dll.a` goes to `lib/`
- if there's no `.dll.a` make sure one is generated. See our policies here: https://github.com/mxe/mxe/wiki/Adding-Shared-Support-to-Packages#pre-commit-checklist
- you could consider using `$(INSTALL) -d` and `$(INSTALL)` rather than `mkdir -p`, `cp` and `chmod`.